content/css/concepts/position/terms/top/top.md

```diff
@@ -43,6 +43,10 @@ Set the position of `.box` element `40px` from the top of the nearest relative p
 }
 ```
 
+![CSS Top Attribute Absolute](images/abs_pos_40px.png)
```

Comment:
```suggestion
![Shows use CSS top attribute with element position set to absolute](images/abs_pos_40px.png)
```
Make the `alt-text` (text inside square brackets) more descriptive, like so. Also, add the updated image link (inside round brackets) in this format `https://raw.githubusercontent.com/Codecademy/docs/main/media/` followed by the image name.